src/gsi/balmod.f90

```diff
@@ -946,9 +947,16 @@ subroutine locatelat_reg(mype)
     allocate( clat_avn(mlat), clat_avn4(mlat) )
     read(lunin)clat_avn4
     close(lunin)
-    do i=1,mlat
-       clat_avn(i)=clat_avn4(i)*deg2rad
-    end do
+    if (usenewgfsberror) then
```

Comment:
From 947 to 949, can it just be like below?
^^
read (lunin) clat_avn
if(.not.usenewgfsberror) clat_avn=clat_avn*degeg2rad
VV
Will this be less readable?

Reply:
Thanks for noticing this. But the clat_avn4 was defined as r_single, but clat_avn was defined as r_kind orginally in GSI. I guess it was designed on purpose for reading the clat with r_single in the BE file.  I prefer keep it as  is. How to you think?
I will change the comment line.

Reply:
Agree! I didn't notice they are of different precisions!